.github/workflows/build_docs.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y inkscape
         sudo apt-get install -y ffmpeg
+        pip install ffmpeg
```

Comment:
nitpick: please add ffmpeg on line 38, after pip has been upgraded and with the other packages being installed